src/MPoly.jl

```diff
@@ -1243,7 +1243,7 @@ If the optional `parent` keyword is provided, the polynomial will be an
 element of `parent`. The caching of the parent object can be controlled
 via the `cached` keyword argument.
 """
-function map_coefficients(f, p::MPolyRingElem; cached = true, parent::AbstractAlgebra.MPolyRing = _change_mpoly_ring(AbstractAlgebra.parent(f(zero(base_ring(p)))), AbstractAlgebra.parent(p), cached))
+function map_coefficients(f, p::MPolyRingElem; parent::AbstractAlgebra.MPolyRing = _change_mpoly_ring(AbstractAlgebra.parent(f(zero(base_ring(p)))), AbstractAlgebra.parent(p), cached))
```

Comment:
I am probably missing something, but isn't `cached` used as the last argument?

Reply:
You are right. I am not sure what kind of drugs I was on when I wrote that *sigh* (well, actually, I do, but... neverminde)